packages/graphback-runtime-mongodb/src/queryBuilder.ts

```diff
@@ -0,0 +1,132 @@
+import {AdvancedFilter} from '@graphback/runtime';
+
+const AND_FIELD = 'and';
+const OR_FIELD = 'or';
+const NOT_FIELD = 'not';
+
+// Operators that work with simple substitution :P
+const operatorMap = {
+    [AND_FIELD]: '$and',
+    [OR_FIELD]: '$or',
+    [NOT_FIELD]: '$nor',
+    ne: '$ne',
+    eq: '$eq',
+    le: '$lte',
+    lt: '$lt',
+    ge: '$gte',
+    gt: '$gt',
+}
+
+type OperatorTransform = [string, any][];
+
+type OperatorTransformFunction = (value: any) => OperatorTransform;
+
+// A map of functions to transform mongodb incompatible operators
+// Each function returns pairs of a key and an object for that key
+const operatorTransform: {
+    [operator: string]: OperatorTransformFunction
+} = {
+    between: (values: [any, any]): OperatorTransform => {
+        values.sort();
+
+        return [
+            [operatorMap.ge, values[0]],
+            [operatorMap.le, values[1]]
+        ]
+    },
+    nbetween: (values: [any, any]): OperatorTransform => {
+
+        values.sort();
+
+        return [
+            [
```

Comment:
Can use operator above

Reply:
This is actually wrong implementation of this operator, fixing it now